modules/langstream/pages/index.adoc

```diff
@@ -0,0 +1,312 @@
+
+Your tenant will be created with a default namespace, which is a logical grouping of topics. You can create additional namespaces to organize your topics.
+
+Your tenant will be listed in the LangStream tab. Select *Enable* to enable LangStream for your tenant.
+
+image::enable-langstream.png[Enable LangStream]
+
+Under the hood, this is enabling the xref:starlight-for-kafka:ROOT:index.adoc[Starlight for Kafka] API for your tenant to connect to your Kafka cluster.
+
+== Build a LangStream Application
+
+Build a LangStream application by creating YAML files to describe the application.
+The application folder structure looks like this:
+
+[source,shell]
+----
+|- sample-app
+|- application
+    |- pipeline.yaml
+    |- gateways.yaml
+|- instance.yaml
+|- configuration.yaml
+|- secrets.yaml
+----
+
+Here's a shortcut:
+[source,shell]
+----
+mkdir sample-app && cd sample-app
+touch configuration.yaml instance.yaml secrets.yaml
+mkdir application && cd application
+touch pipeline.yaml gateways.yaml
+----
+
+Next, you will populate the YAML files to connect your application to your Astra Streaming tenant.
+
+== Populate YAML files
+
+Instance.yaml declares the application's processing infrastructure, including where streaming and compute take place.
+The values for the Kafka bootstrap server are found in your Astra Streaming tenant, in the Starlight for Kafka ssl.properties file.
+The secrets for tokens and passwords are stored in the secrets.yaml file, which you'll populate in the next step.
+[source,yaml]
+----
+instance:
+  streamingCluster:
+    type: "kafka"
+    configuration:
+      admin:
+        bootstrap.servers: kafka-gcp-useast1.streaming.datastax.com:9093
+        security.protocol: SASL_SSL
+        sasl.jaas.config: "org.apache.kafka.common.security.plain.PlainLoginModule required username='{{ secrets.astra-token.tenant }}' password='token:{{ secrets.astra-token.pulsar-token }}';"
+        sasl.mechanism: PLAIN
+        session.timeout.ms: 45000
+
+  computeCluster:
+    type: "kubernetes"
+----
+
+Secrets.yaml contains auth information for connecting to other services.
+These values are referenced with https://mustache.github.io/mustache.5.html[Mustache templating^]{external-link-icon} - for example, the Astra tenant name is referenced in instance.yaml with `'{{ secrets.astra-token.tenant }}'`.
+The pulsar-token is the Astra Streaming token, which is generated when you create a tenant.
+The Azure access key and URL are found in your Azure deployment.
+For more on finding secrets, see https://docs.langstream.ai/building-applications/secrets.html[Secrets^]{external-link-icon}.
+[source,yaml]
+----
+secrets:
+  - name: astra-token
+    id: astra-token
+    data:
+      pulsar-token: eyJhbGc...
+      tenant: langstream-tenant
+      namespace: kafka
+  - name: open-ai
+    id: open-ai
+    data:
+      access-key: 783f...
+      url: https://company-openai-dev.openai.azure.com/
```

Comment:
add provider (see the latest version of the example code), with default to 'openai'

and add "embeddings-model" and "chat-completions-model",
they are to be customiser for each user for sure